### PR TITLE
Version 6.8.1

### PR DESCRIPTION
**Pull request by vladrusu**
- Improvements to url parsing: make sure do not touch [url] or [img] tags
- Also linkify www.domain.com (without http://)
- Added mailto protocol & support for [url="mailto:test@email.com"]email[/url]
- Support for dash in domain name (www.my-domain.com)

### CODE REVIEW

File: src/Decoda/Hook/ClickableHook.php [hunk 1]
```diff
@@ -23,29 +23,53 @@ class ClickableHook extends AbstractHook {
     public function beforeParse($content) {
         $parser = $this->getParser();
 
+        // To make sure we won't parse links inside [url] or [img] tags, we'll first replace all urls/imgs with uniqids
+        // and keep them in this array, and restore them at the end, after parsing
+        $ignoredStrings = array();
+
+        // The tags we won't touch
+        // For example, neither [url="http://www.example.com"] nor [img]http://www.example.com[/img] will be replaced.
+        $ignoredTags = array('url', 'img');
```
Comment (milesj) [inline src/Decoda/Hook/ClickableHook.php]: There's also `image` and `link`.

File: src/Decoda/Hook/ClickableHook.php [hunk 2]
```diff
@@ -23,29 +23,53 @@ class ClickableHook extends AbstractHook {
...
+
+        $i = 0;
+        foreach ($ignoredTags as $tag) {
+	        if (preg_match_all(sprintf('/\[%s[=\]].*?\[\/%s\]/is', $tag, $tag), $content, $matches, PREG_SET_ORDER)) {
```
Comment (milesj) [inline src/Decoda/Hook/ClickableHook.php]: Indentation is off.

Comment (vladrusu) [inline src/Decoda/Hook/ClickableHook.php]: Fixed both issues (image/link tags and identation), I updated the file.

**Comment by milesj** [comment]
Looking good.

Not exactly related to this PR, but can you take a look at this: https://github.com/milesj/decoda/pull/125 Seems to be a regression from the last version.

**Comment by vladrusu** [comment]
Hello !
I'll run that test in the next few minutes and see what happens. :)

**Comment by vladrusu** [comment]
I can confirm that #125 is fixed in 6.8.1.
I added a test case in ClickableHookTest.php and all seems ok.

**Comment by milesj** [comment]
Looks good. I'll merge and tag this in a bit.

**Pull request merged**